README.md

```diff
@@ -108,19 +108,25 @@ This weirdness is tracked in [issue 100](https://github.com/include-what-you-use
 
 The original design was built for Make, but a number of alternative run modes have come up over the years.
 
-#### Plugging into Make ####
+#### Running on single source file ####
 
-The easiest way to run IWYU over your codebase is to run
+The simplest way to use IWYU is to run it against a single source file:
 
-      make -k CXX=/path/to/llvm/Debug+Asserts/bin/include-what-you-use
+      include-what-you-use $CXXFLAGS myfile.cc
 
-or
+where `$CFLAGS` are the flags you would normally pass to the compiler.
 
-      make -k CXX=/path/to/llvm/Release/bin/include-what-you-use
+#### Plugging into existing build system ####
 
-(include-what-you-use always exits with an error code, so the build system knows it didn't build a .o file.  Hence the need for `-k`.)
+The simplest example above doesn't scale very well to projects with many source files, so typically you would replace the compiler in your build system with `include-what-you-use` to generate a large batch of IWYU advice.
```

Comment:
I don't like this advice very much.  It scales well for me on a project with a few hundred source files, and I think it's the ideal usage, so I wouldn't discourage about it.  What is true is that it doesn't integrate well into _existing_ build systems (i.e., if you're not redesigning your build system to use iwyu(1), but instead just amend it to add this linter)

Reply:
See other comment. It's generally hard to conjure the compiler flags from nowhere. The build system is the only source of truth. There may be projects where the flags are very few, or very consistent between modules, but that's usually not the norm with the kinds of projects interested in using IWYU for legacy cleanup. Again, I agree that there's probably better wording :-)

Reply:
Yup, in the project I'm talking about, I also have different CFLAGS for each module.  But the same way I can differentiate them for `CC` and `CXX` with a few makefile rules/variables, I can reuse the same to make iwyu(1) work properly.

If you're curious, you can read this fragment of a Makefile: <http://www.alejandro-colomar.es/src/alx/alx/libalx.git/tree/lib/recipes_build.inc.mk>
I still haven't published code for using `iwyu`, but you can see the code for running clang-tidy and other linters, which are almost identical.

Reply:
Softened the language here

Reply:
I like it now :)